plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/helpers.ts

```diff
@@ -30,6 +32,10 @@ export async function getOctokitOptions(options: {
 }): Promise<OctokitOptions> {
   const { integrations, credentialsProvider, repoUrl, token } = options;
   const { owner, repo, host } = parseRepoUrl(repoUrl, integrations);
+  const requestOptions = {
+    // set timeout to 60 seconds
+    timeout: 60 * SECOND,
```

Comment:
Maybe we can do this as `60_000` instead of declaring a const. It's what we've done in other parts of the project at least :)

Reply:
so a `const DEFAULT_TIMEOUT_MS` at the top or so?

Reply:
Ah cool! Cheers @benjdlambert

Reply:
Just missed your comment @freben. I'll add in the const